task03.rb

```diff
@@ -0,0 +1,9 @@
+#[#3] Task03 - For a given array that contains the string "foo" - output it at the end without changing the places of the other items in the array, e.g. [1, :here, nil, "some", "foo", "thing", ""] should output [1, :here, nil, "some", "thing", "", "foo"] 
+
+def task03(arr)
+    if arr.nil? || arr.empty?
```

Comment:
@MaraEfimova There are quite a lot of invalid inputs like Ranges, Integers, Floats, etc.